src/ale/4C_ale_ale2.hpp

```diff
@@ -54,7 +54,7 @@ namespace Discret
       void nodal_block_information(Core::Elements::Element* dwele, int& numdf, int& dimns) override;
 
       Core::LinAlg::SerialDenseMatrix compute_null_space(
-          Core::Nodes::Node& node, const double* x0, const int numdof, const int dimnsp) override;
+          Core::Nodes::Node& node, std::span<const double, 3> x0, const int numdof) override;
```

Comment:
I don't think it makes a lot of sense to hard-code this to dim `3`. With #1283, we will have correct dynamic coordinate numbers. This span should probably be


```suggestion
          Core::Nodes::Node& node, std::span<const double> x0, const int numdof) override;
```

Ofc, you can can assert inside the derived classes that the number of coordinates is as you expect.